pycat/esd/methods.py

```diff
@@ -162,9 +162,9 @@ def relative_sdm(
             sce_gamma = gamma.fit(sce_raindays, floc=0)
 
             expected_sce_raindays = min(
-                np.round(
```

Comment:
Thanks a lot. Indexing with floats was deprecated in numpy 1.12. See [release notes](https://github.com/numpy/numpy/releases?after=v1.12.0).